src/cluster_legacy.h

```diff
@@ -221,7 +221,8 @@ typedef struct {
     char slaveof[CLUSTER_NAMELEN];
     char myip[NET_IP_STR_LEN];    /* Sender IP, if not all zeroed. */
     uint16_t extensions; /* Number of extensions sent along with this packet. */
-    char notused1[30];   /* 30 bytes reserved for future usage. */
+    uint8_t extensions_supported; /* Does the node support parsing ping extensions. */
```

Comment:
Still not sure why we wouldn't have this be generic flags instead of using 8 bits for 1 value.